app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_list = command.split()
+    command_name = command_list[0]
+    file_to_remove = command_list[1]
+    new_file = command_list[2]
+
+    if command_name == "mv":
+
+        if "/" not in command_list[2] and file_to_remove != new_file:
```

Comment:
use ```new_file``` instead of ```command_list[2]```

Reply:
@OleksandrPozden 

Please, have a look at the "Solution after changes", [b0525cf](https://github.com/mate-academy/py-move-file/pull/131/commits/b0525cfe69cfa7fb1f788a129cfba7a31520da03)